src/main/webapp/i18n/de/user-management.json

```diff
@@ -1,5 +1,12 @@
 {
     "userManagement": {
+        "batch": {
+            "all": "Alle",
+            "delete": {
+                "question": "Sollen die Nutzer:innen ( <strong>{{ title }}</strong> ) wirklich dauerhaft gelöscht werden? Diese Aktion kann NICHT rückgängig gemacht werden!",
+                "typeNumberToConfirm": "Bitte gib die Anzahl der Nutzer:innen zur Bestätigung ein."
```

Comment:
Typically the expected value is also included in the text. I also don't think that in the use case of selecting some filters and deleting the whole result someone wants to count the number by themselves.